### ISSUE TITLE
Question: Why aren't field options available to templates?

### ISSUE THREAD

**Issue by ThisIsMissEm** [thoughtbot/administrate#612]
In https://github.com/thoughtbot/administrate/blob/master/lib/administrate/field/base.rb#L46 `options` is marked as protected, which seems to mean that from customised views, I couldn't make use of the options. 

My use case was to add an option to `BelongsTo` and customise the views as to allow sometimes not having a `BelongsTo` as a link (we have some cases where we don't actually want an admin panel for certain resources, but `BelongsTo` expects there to be one).

For now I've just worked around this by creating a `TextualBelongsTo` which subclasses from `BelongsTo` and just changes the templates a bit. It seems to be nicer to actually just be able to specify arbitrary options and then just customise the views, rather than subclassing.

**Comment by j-dexx** [comment]
You can use it in your subclass, for example I have a subclass that has the following public method (so I can then use it in views)

```
def thumbnail                                                  
  options.fetch(:thumbnail, false)                       
end 
```

**Comment by carlosramireziii** [comment]
@ThisIsMissEm Just checking in on this...

Was @j-dexx's code snippet useful to you for solving your problem? I agree with the approach-- you can create subclasses of fields and expose public methods to use in your views. Does that approach work for you?

If not, what kind of behavior would you like to see? If you could illustrate with an example (pseudocode is fine) that would be helpful. 

Let us know, thanks!

**Comment by ThisIsMissEm** [comment]
@carlosramireziii I guess I could do it with subclasses, but then it's a question of [batteries included or bring your own batteries](https://www.quora.com/What-does-batteries-included-philosophy-mean/answer/Nas-Banov)

Is there any negative side effect of exposing all the options to the View templates?

**Comment by pablobm** [comment]
Also raised at https://github.com/thoughtbot/administrate/issues/1202

**Comment by pablobm** [comment]
Fixed by #1513
